src/AstronautsTable.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import fetchJson from "./fetchJson";
+
+function AstronautsTable() {
```

Comment:
I guess I'll leave it as a question for Charles or Tony, but I'm used to functions being `camelCased` not `titleCased`.

Reply:
When I changed it, I got this error:
```
  Line 6:29:  React Hook "useState" is called in function "astronautsTable" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use" 
```

So I guess because it's a component it needs an uppercase letter in the conventions of React

Reply:
Ahhh, got it. Thanks, I did wonder if this was a React thing and it seems like it is!

Reply:
@hnlee it is a functional react component. So `const ComponentName = props => {}` would be correct. However, `displayData` should be camel-cased.  @graceolivia we need to get you set up with prettier and get your react linted.

Reply:
We should look at breaking out components and passing props. Instead of having a `.map()` that iterates through and generates HTML. an example would be `<AstronautRow name={info.name} craft={info.craft} />`. You can then output components from the map directly. All of your styling would then be kept inside the new row component. 

A rough outline would like this:
```
App 
|-Astronaut Table Component
|--Header Component - pass in the total count of astronauts and display that in your header)
|.                                       so your 
|--Astronaut Row Components - pass in name and craft, render the components out of the map.
```

There shouldn't be any HTML inside the table component itself. It should all be controlled with child components.

The next step, and this would be completely optional would be to pull your query up to the App so that the data could be passed down between multiple child components. This would allow you to pass dynamic data into your header and pull it out of the table component. This way your header component only needs to exist one time and you could conditionally render other tables / information without having to duplicate the code. This might be something to attempt after you get filtering down.